internal/gojsonschema/schemaLoader_test.go

```diff
@@ -34,7 +34,7 @@ func TestSchemaLoaderWithReferenceToAddedSchema(t *testing.T) {
 	assert.Nil(t, err)
 	result, err := schema.Validate(NewStringLoader(`"hello"`))
 	assert.Nil(t, err)
-	if len(result.Errors()) != 1 || result.Errors()[0].Type() != "invalid_type" {
+	if len(result.Errors()) != 1 || result.Errors()[0].Type() != errInvalidType {
```

Comment:
.... ah, because this one is used here. Hmm. 🤔 Does that seem convincing to you?

Reply:
I agree - it's weird to only do it for one and not the others.

I think the better way would be to use the new `errors.Is()` - which would make a lot of this error identification/handling _much_ cleaner. I plan on going back and making a PR with the new 1.13 error features which would make this go away. I'm happy to revert (and turn off goconst for this file).

Reply:
Let's turn goconst off then. I'd love to see `errors.Is` applied more widely to the codebase 😃

Reply:
Sounds good! Done.